python-checks/src/test/java/org/sonar/python/checks/quickfix/PythonQuickFixVerifierTest.java

```diff
@@ -109,7 +121,22 @@ public void initialize(Context context) {
       });
     }
   }
+  private class SimpleCheckInvalidSyntax extends PythonSubscriptionCheck {
```

Comment:
I don't think you need this special made quick fix to create the syntax error, since the mere fact that the expected fixed code contains a syntax error will fail the test, without checking what was actually applied.